normandy/settings.py

```diff
@@ -116,6 +124,11 @@ class Production(Base):
     """Settings for the production environment."""
     STATICFILES_STORAGE = 'whitenoise.django.GzipManifestStaticFilesStorage'
 
+    DEFAULT_FILE_STORAGE = 'storages.backends.s3boto.S3BotoStorage'
+    AWS_ACCESS_KEY_ID = values.SecretValue()
```

Comment:
I read this more closely, and I realize now this is the Production config. Making these required sounds like a good thing in that case.